test/Microsoft.AspNetCore.Mvc.TagHelpers.Test/DistributedCacheTagHelperTest.cs

```diff
@@ -777,6 +778,19 @@ public void UpdateCacheEntryOptions_SetsSlidingExpiration_IfExpiresSlidingIsSet(
                     return Task.FromResult<TagHelperContent>(tagHelperContent);
                 });
 
+            TagHelperOutput tagHelperOutput2 = new TagHelperOutput(
```

Comment:
:clap: Definitely `true` that two workers can never get the same `TagHelperOutput` in real life.